packages/in_app_purchase/in_app_purchase_android/test/billing_client_wrappers/sku_details_wrapper_deprecated_test.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:in_app_purchase_android/billing_client_wrappers.dart';
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// TODO(mvanbeusekom): Remove this file when the deprecated
+//                     `SkuDetailsWrapper.introductoryPriceMicros` field is
+//                     removed.
+
+void main() {
+  test(
+      'Deprecated `introductoryPriceMicros` field reflects parameter from constructor',
+      () {
+    final SkuDetailsWrapper skuDetails = SkuDetailsWrapper(
+      description: 'description',
+      freeTrialPeriod: 'freeTrialPeriod',
+      introductoryPrice: 'introductoryPrice',
+      // ignore: deprecated_member_use_from_same_package
+      introductoryPriceMicros: '990000',
+      introductoryPriceCycles: 1,
+      introductoryPricePeriod: 'introductoryPricePeriod',
+      price: 'price',
+      priceAmountMicros: 1000,
+      priceCurrencyCode: 'priceCurrencyCode',
+      priceCurrencySymbol: r'$',
+      sku: 'sku',
+      subscriptionPeriod: 'subscriptionPeriod',
+      title: 'title',
+      type: SkuType.inapp,
+      originalPrice: 'originalPrice',
+      originalPriceAmountMicros: 1000,
+    );
+
+    expect(skuDetails, isNotNull);
+    expect(skuDetails.introductoryPriceAmountMicros, 0);
+    // ignore: deprecated_member_use_from_same_package
+    expect(skuDetails.introductoryPriceMicros, '990000');
+  });
+
+  test(
+      '`introductoryPriceAmoutMicros` constructor parameter is reflected by deprecated `introductoryPriceMicros` and `introductoryPriceAmountMicros` fields',
+      () {
+    final SkuDetailsWrapper skuDetails = SkuDetailsWrapper(
+      description: 'description',
+      freeTrialPeriod: 'freeTrialPeriod',
+      introductoryPrice: 'introductoryPrice',
+      introductoryPriceAmountMicros: 990000,
+      introductoryPriceCycles: 1,
+      introductoryPricePeriod: 'introductoryPricePeriod',
+      price: 'price',
+      priceAmountMicros: 1000,
+      priceCurrencyCode: 'priceCurrencyCode',
+      priceCurrencySymbol: r'$',
+      sku: 'sku',
+      subscriptionPeriod: 'subscriptionPeriod',
+      title: 'title',
+      type: SkuType.inapp,
+      originalPrice: 'originalPrice',
+      originalPriceAmountMicros: 1000,
+    );
+
+    expect(skuDetails, isNotNull);
+    expect(skuDetails.introductoryPriceAmountMicros, 990000);
+    // ignore: deprecated_member_use_from_same_package
+    expect(skuDetails.introductoryPriceMicros, '990000');
+  });
+}
```

Comment:
Let's add a setter test to cover the missed breaking change. It's fine if it's testing that it's a no-op, it just needs to build.

Reply:
As per previous comment, I don't think setter is necessary after all.